src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -122,7 +122,7 @@ public void set(double speed) {
   /**
    * Stops the motors
    */
-  public void stopShooter() {
+  public void stop() {
```

Comment:
Can we rename this back to `stopShooter()`? Since we've got both a shooter and a hood in this subsystem, I think it makes sense to have them both be named explicitly.

Side note: Remember to test that your changes still build. Elsewhere in the code, we're still calling `stopShooter()`, so this change breakes the code.